packages/esm-patient-test-results-app/package.json

```diff
@@ -33,11 +33,11 @@
     "url": "https://github.com/openmrs/openmrs-esm-patient-chart/issues"
   },
   "dependencies": {
-    "@carbon/charts-react": "^0.41.14",
+    "@carbon/charts-react": "^0.49.4",
```

Comment:
I'd suggest upgrading *every* charts/d3 version accross the entire monorepo at once to avoid version inconsistencies. The two libs are referenced in > 10 repos.

Note: I've also done these changes [here](https://github.com/openmrs/openmrs-esm-patient-chart/pull/393) to fix issues with the trendline charts. It may not be necesary to make the same changes in this PR too.